content/integrations/bitbucket.md

```diff
@@ -26,21 +26,28 @@ If you view a dashboard, in the top left search bar you can type ```sources:bitb
 
 ## Installation
 
-On the Bitbucket website, create a new user that has **Read** access to the repo you want to monitor. This is the user that Datadog will use to collect information on the repository.
+Setup your webhooks on the bitbucket side. [Instructions here][3].
```

Comment:
This could be more descriptive:

"See [Bitbucket's documentation] and set up webhooks for any Bitbucket behaviors you want to track in Datadog."